shell/Gemfile

```diff
@@ -2,4 +2,5 @@ source 'https://rubygems.org'
 
 gem 'pry', '~> 0.10.1'
 gem 'netrc', '~> 0.7.7'
-gem 'looker-sdk',  :git => 'git@github.com:looker/looker-sdk-ruby.git'
+# gem 'looker-sdk',  :git => 'git@github.com:looker/looker-sdk-ruby.git'
+gem 'looker-sdk',  :path => '../'
```

Comment:
Well, arguably, this relative path is better. The alternative will be the normal version number constraint string as above after the sdk is registered with rubygems. But, since this is part of that sdk, the relative path is starting to look like a winner to me.